### PR TITLE
fix(quickAdd): open detail page on "Add and Open"

### PR DESCRIPTION
**Pull request by sahil143**
### [ ] What does this PR do? _[Mandatory]_
Clicking on "Add and Open" : 
 - Adds the WorkItem
- Open the detail page

### [ ] What issue/task does this PR references? _[Mandatory]_

https://github.com/openshiftio/openshift.io/issues/2995

### [ ] Are the tests Included? _[Mandatory]_

Yes, [here](https://github.com/fabric8-ui/fabric8-planner/pull/2578)

### CODE REVIEW

**Comment by fabric8cd** [comment]
@sahil143 fabric8/fabric8-ui:SNAPSHOT-PR-2579-2 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2579-fabric8-planner.badger.fabric8.io

File: src/tests/specs/smoke/smokeTest.spec.ts [hunk 1]
```diff
@@ -159,8 +159,10 @@ describe('Planner Smoke Tests:', () => {
     await planner.quickPreview.close();
   });
 
-  it('Create a work item and Open detail page', async() => {
+  fit('Create a work item and Open detail page', async() => {
```
Comment (jarifibrahim) [inline src/tests/specs/smoke/smokeTest.spec.ts]: Please remove `fit`

**Comment by fabric8cd** [comment]
@sahil143 fabric8/fabric8-ui:SNAPSHOT-PR-2579-3 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2579-fabric8-planner.badger.fabric8.io

**Comment by nimishamukherjee** [comment]
[test]

**Comment by nimishamukherjee** [comment]
[test]

**Pull request merged**